app/views/plans/edit.html.erb

```diff
@@ -0,0 +1,93 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-offset-2 col-md-8">
+      <div class="page-header">
+        <h3>Edit Plan</h3>
+        <%= plan_path(@plan) %>
+      </div>
+      <%= form_for(@plan, url: plan_path(@plan) ,method: :put, :html => { :multipart => true }) do |f| %>
```

Comment:
The comma before `method: :put` has a space before and none after. While you're at it, consider switching hashes to the new syntax, *e.g.*:
```ruby
:html => { :multipart => true }
:required => true
```
Should be changed to:
```ruby
html: { multipart: true }
required: true
```